src/components/Drawer/AddReportDrawer/AddReportDrawerContent.tsx

```diff
@@ -25,13 +22,17 @@ import poiAddDrawerTargetCategory from "../../../assets/images/poiAddDrawerTarge
 import poiAddDrawerTargetName from "../../../assets/images/poiAddDrawerTargetName.svg";
 import poiAddDrawerUploadImages from "../../../assets/images/poiAddDrawerUploadImages.svg";
 
-const FloorSelect: React.FC<{ cluster: Cluster | null }> = ({ cluster }) => {
+import floorSelctions from "../../../assets/data/options/selections/floor.json";
+import targetCategorySelections from "../../../assets/data/options/selections/target-category.json";
+import targetNameSelections from "../../../assets/data/options/selections/target-name.json";
+
+const FloorSelect: React.FC = () => {
   const { t } = useTranslation();
 
   const dispatch = useDispatch();
   const reportData = useSelector((state: IRootState) => state.report.data);
   const floor = reportData.floor;
-  const floorOptions = getOptions(cluster?.data.name || "").floor;
+  const floorOptions = floorSelctions;
```

Comment:
Why need to assign the data here? Maybe use `floorSelectoins` directly is better. Same as `targetCategorySelections` & `targetNameSelections`.

Reply:
Ok, I modified the code.